### ISSUE TITLE
Error on Mac OSX

### ISSUE THREAD

**Issue by pedrosimao** [TitorX/Postdown#3]
I am running on Mac OSX environment and I am getting the following error:
```
Traceback (most recent call last):
  File "/usr/local/anaconda3/bin/postdown", line 11, in <module>
    sys.exit(execute())
  File "/usr/local/anaconda3/lib/python3.6/site-packages/postdown/cmdline.py", line 14, in execute
    parse(in_file, out_file)
  File "/usr/local/anaconda3/lib/python3.6/site-packages/postdown/parser.py", line 28, in parse
    request = api['request']
KeyError: 'request'
```

**Comment by TitorX** [comment]
![image](https://user-images.githubusercontent.com/11922492/38597088-14ec89b6-3d88-11e8-8c3a-079350829f94.png)

It's only support **Collection v2**

You should check if your json file is right version.

or, paste your json file down there. 
I will fix it.

**Comment by pedrosimao** [comment]
Hey, I am pretty sure it is version 2.
Here is a link to the file:
https://drive.google.com/open?id=1CvCbWW1FnmlTBJ_P2kKcKPivs7c8W180

Thanks for your help!

**Comment by TitorX** [comment]
I figured out what happened.

![image](https://user-images.githubusercontent.com/11922492/38653234-5897462e-3e3c-11e8-9fc4-1f767225d9e7.png)

You created a subdirectory in collection. This error was caused by subdirectory.

I didn't write this feature into README, actually, it only support the collection which directly put apis down the collection.

I mean, like this
![image](https://user-images.githubusercontent.com/11922492/38653387-11c765d4-3e3d-11e8-9aa7-ec6e1d732625.png)

No subdirectory.

I already tested your json file in OSX. Successful.
